WordPress/src/main/res/values/styles.xml

```diff
@@ -518,9 +518,6 @@
         <item name="android:layout_height">@dimen/my_site_list_row_icon_size</item>
         <item name="android:layout_alignParentStart">true</item>
         <item name="android:layout_centerVertical">true</item>
-        <item name="android:layout_marginTop">@dimen/margin_large</item>
-        <item name="android:layout_marginBottom">@dimen/margin_large</item>
-        <item name="android:layout_marginEnd">@dimen/my_site_list_row_icon_margin_right</item>
```

Comment:
I believe some of these styling changes have caused the My Site item layout to break when using the app with the feature flag turned off:

<img width=300 src="https://user-images.githubusercontent.com/830056/105773708-197cf800-5f43-11eb-9fa9-82fe71eeb197.png"/>

Reply:
I actually just realized this also affects the bottom sheet's items layout:

<img width=300 src="https://user-images.githubusercontent.com/830056/105776801-002a7a80-5f48-11eb-9cb1-ed775556c937.png"/>

<img width=300 src="https://user-images.githubusercontent.com/830056/105776808-01f43e00-5f48-11eb-83f0-be092c52c1f5.png"/>

Reply:
Good catch! I've reverted the style changes and created a copies called "Updated.."